specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ContainerApps_ListBySubscription.json

```diff
@@ -16,7 +16,7 @@
             "properties": {
               "provisioningState": "Succeeded",
               "managedEnvironmentId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/demokube",
-              "workloadProfileType": "GeneralPurpose",
+              "workloadProfile": "GeneralPurpose",
```

Comment:
Values for `workloadprofile` to be same as examples for consistency i.e. `My-GeneralPurpose`